openedx/core/djangoapps/discussions/models.py

```diff
@@ -48,6 +59,12 @@ class LearningContextDiscussionConfig(models.Model):
     config_slug = models.CharField(
         blank=False,
         max_length=100,
-        verbose_name=_("Unique configuration slug"),
-        help_text=_("A unique identifier for this configuration."),
+        verbose_name=_("Configuration slug"),
+        help_text=_("Should be an existing slug from a DiscussionConfig"),
```

Comment:
Should it be a ForeignKey, if it needs to match an existing field?

Reply:
The issue with a foreign key is that it can only link to an single instance, by doing it though a slug we reference a different non-unique id. So, a course admin can change this configuration, and the latest configuration will be applied without needing to change each course that uses that config.